vmdb/spec/tools/fix_auth/models_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "spec_helper"
+
+$LOAD_PATH << Rails.root.join("tools")
+
+require "fix_auth"
+require "tempfile"
+require "yaml"
+
+describe FixAuth::FixDatabaseYml do
+  subject { described_class }
+
+  # TODO: add legacy password tests
+
+  it "updates password" do
+    with_databaseyml("oldpassword") do |databaseyml|
+      FixAuth::FixDatabaseYml.file_name = databaseyml
+      FixAuth::FixDatabaseYml.run(:hardcode => 'newpassword')
+
+      expect(dbpassword(databaseyml)).to be_encrypted("newpassword")
+    end
+  end
+
+  it "doesnt add a password" do
```

Comment:
if you're in here to fix the seemingly unused `subject`, maybe change this to "does not".